src/components/ChatBotMessages/index.scss

```diff
@@ -0,0 +1,11 @@
+// ChatBot inhereted class
+.rsc-os-option-element {
+  margin-left: 2rem;
+}
+
+a {
```

Comment:
this is affecting all `a` in the app, isn't it?

Reply:
I am not sure, but I put it in class to prevent any issues